### PR TITLE
Updated docs styles

### PR DESCRIPTION
**Pull request by brianjo**
Updated requirements.txt and conf.py.

### CODE REVIEW

**Comment by SsnL** [comment]
Need a rebase

**Review by ericnakagawa — APPROVED** [review]
All internal tests pass. Thanks Brian.

File: docs/source/conf.py [hunk 1]
```diff
@@ -134,19 +135,13 @@
 #
 #
 #
-html_theme = 'sphinx_rtd_theme'
 
+html_theme = 'pytorch_sphinx_theme'
+html_theme_path = [pytorch_sphinx_theme.get_html_theme_path()]
 
 # Theme options are theme-specific and customize the look and feel of a theme
```
Comment (SsnL) [inline docs/source/conf.py]: This comment seems obsolete.

**Review by ssnl — COMMENTED** [review]
Local build is busted. It succeeds, but when viewing, I'm getting a lot of http errors, e.g.,
```
127.0.0.1 - - [19/Sep/2018 07:55:59] "GET /build/html/_static/images/chevron-right-orange.svg HTTP/1.1" 304 -
127.0.0.1 - - [19/Sep/2018 07:55:59] code 404, message File not found
127.0.0.1 - - [19/Sep/2018 07:55:59] code 404, message File not found
```

I installed the dependencies and used the usual build command `make clean; make html`.

**Comment by SsnL** [comment]
Here's how local build looks like for me.

screenshot 2018-09-19 10 57 32
screenshot 2018-09-19 10 57 40

**Comment by SsnL** [comment]
After cmd+shift+r the website loads! Thanks @brianjo . However, it seems that the declarations are all rendered wrongly, e.g.,
```
torch.set_printoptions(precision=None, threshold=None, edgeitems=None, linewidth=None, profile=None)
```
looks like
screenshot 2018-09-19 11 15 07

and
```
torch.distributed.send(tensor, dst, group=<object object>, tag=0)
```
looks like
screenshot 2018-09-19 11 15 44

**Review by zou3519 — CHANGES_REQUESTED** [review]
Requesting changes based on @SsnL's comments (website not rendering correctly locally)

**Comment by SsnL** [comment]
@zou3519 Yes, this seems like a theme-related issue. @brianjo is actively working with the vendors to resolve this. Hopefully we can get this nice looking theme merged soon. :)

**Comment by ericnakagawa** [comment]
Folks, the theme issue has been resolved.

screen shot 2018-09-20 at 4 26 49 pm

Please merge.

**Pull request closed without merging**